qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -219,9 +231,6 @@
             cmd_get_disk = cat /proc/mounts |awk '$2~/^\%s$/{print $1,$3}'
             Windows:
                 cmd_get_disk = wmic volume where "DriveLetter='%s'" get FileSystem,DeviceID |findstr /v /i FileSystem
-        - check_nonexistent_cmd:
```

Comment:
This is a new cases that be added few days ago, could you explain why did you delete it?

Reply:
https://github.com/autotest/tp-qemu/pull/1876

Reply:
@yanan-fu I'm not sure the reason, maybe misoperation or rebase issue.
Anyway ,add this config part again.
thanks.